decidim-meetings/spec/lib/meetings/registrations/code_generator_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Meetings
+    module Registrations
+      describe CodeGenerator do
+        subject { described_class.new(length: length) }
+
+        let(:length) { 6 }
+        let(:meeting) { build(:meeting) }
+        let(:registration) { build :registration, meeting: meeting }
+
+        describe "#generate" do
+          let(:existing_code) { "AS35TY58" }
+          let(:valid_code) { "QW89HJ34" }
+          let(:code) { subject.generate(registration) }
+
+          before do
+            create :registration, meeting: meeting, code: existing_code
+
+            expect(subject)
+              .to receive(:choose)
+              .with(length)
+              .exactly(2).times
+              .and_return(existing_code, valid_code)
```

Comment:
@mrcasals, @deivid-rodriguez I need help testing this. Rubocop complains with `RSpec/SubjectStub: Do not stub your test subject`, can you help me with another way to test this method?

Class generating the code:  https://github.com/decidim/decidim/pull/3805/files#diff-6c225d57a5e2477b37660748966d7262R18

Reply:
Test looks good and readable to me, I would disable that cop for the test.

Reply:
Thanks to both, I agree with your comments, so I'll disable the cop for the test.